revolv/static/project.scss

```diff
@@ -439,7 +439,7 @@ $gray-9: #999999;
             }
         }
 
-        .update-text {
+        .update-text p {
```

Comment:
couldn't you just delete this? i searched project.html but there are no items with class update-text that are of type p